### DISCUSSION TITLE
Documentation is not updated on ccache.dev site

### DISCUSSION THREAD

**Discussion by dyadav7** [ccache/ccache#627] [Support]
### Question ###

Why https://ccache.dev/manual/3.7.11.html#_sharing_a_cache_on_nfs does not show changes done in https://github.com/ccache/ccache/pull/604

https://github.com/ccache/ccache/blob/master/doc/MANUAL.adoc#sharing-a-cache-on-nfs
Shows the https://github.com/ccache/ccache/pull/604 changes.

**Comment by jrosdahl** [comment]
That's because the 3.7.x releases are made from the 3.7-maint branch but #604 targeted master which is the development branch for the next non-bug fix release (currently targeting version 4.0). More info here: https://ccache.dev/repo.html

I usually backport changes like this to 3.7-maint but I apparently didn't do that for #604. ... But done in 3b4b514f58ba and 0d0f48375aa6 now, so they will be part of the 3.7.12 documentation if and when 3.7.12 will be released.

**Comment by dyadav7** [comment]
I understand it now.
Thanks a lot for back porting.
